src/api/spec/models/attrib_spec.rb

```diff
@@ -62,4 +62,18 @@
       end
     end
   end
+
+  context "#project" do
+    context "attribute with project" do
+      let(:attribute_with_project) { create(:attrib, project: project) }
+
+      it { expect(attribute_with_project.project).to eq(project) }
```

Comment:
No. Here it is being tested the `project` method, without any call to `container` method.